### PR TITLE
feat: add support for 4090 series (US - b&h/newegg/bestbuy)

### PR DESCRIPTION
**Pull request by mrmos**
### Description

adds support for all available 4090 models on bh + newegg + bestbuy. 

can add more stores as needed, but wanted to get the ball rolling. 

new contributor to this project, so let me know if there's anything I missed. 

Thanks!

### CODE REVIEW

**Comment by Mr-Tech-13** [comment]
https://github.com/Mr-Tech-13/4090-tracker This is my fork, updated for 4080 just today. Feel free to add this to your code!

**Review by jef — APPROVED** [review]

File: docs/reference/filter.md [hunk 1]
```diff
@@ -262,6 +263,7 @@ Used with the `SHOW_ONLY_SERIES` variable.
 | Nvidia RTX 3080 | `3080` |
 | Nvidia RTX 3080 Ti | `3080ti` |
 | Nvidia RTX 3090 | `3090` |
+| Nvidia RTX 3090 | `4090` |
```
Comment (jef) [inline docs/reference/filter.md:266]:
```suggestion
| Nvidia RTX 4090 | `4090` |
```

Comment (jef) [reply docs/reference/filter.md:266]: Once you update this @mrmos, we can merge this in! Thanks for your contribution :)

Comment (jef) [reply docs/reference/filter.md:266]: Actually I'm going to go ahead and merge this then update myself.

Thank you!

Comment (mrmos) [reply docs/reference/filter.md:266]: Thanks jef!

**Comment by jhs232** [comment]
Sorry i never got back to you earlier. Here are all the store files I edited to add the 4090s (over a month ago though, so probably not quite up to date).
[Stores.zip](https://github.com/jef/streetmerchant/files/10164773/Stores.zip)

**Pull request merged**